README.md

```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
```

Comment:
Note that you don't have to number the items yourself. What I mean is this:

```
1. Firm Dynamics and Innovation
1. Analysis of Micro Data
1. This Class
```

Renders like this:

1. Firm Dynamics and Innovation
1. Analysis of Micro Data
1. This Class